### PR TITLE
UEFI cleanup and some fixing.

### PR DESCRIPTION
**Pull request by liba2k**

### CODE REVIEW

File: qiling/os/uefi/utils.py [hunk 1]
```diff
@@ -18,4 +20,20 @@ def check_and_notify_protocols(ql):
...
         return True
-    return False
\ No newline at end of file
+    return False
+
+def write_int32(ql, address, num):
+    if ql.archendian == QL_ENDIAN.EL:
+        ql.mem.write(address, struct.pack('<I',(num)))
+    else:
+        ql.mem.write(address, struct.pack('>I',(num)))
```
Comment (aquynh) [inline qiling/os/uefi/utils.py:29]: please put blank lines between functions (double-lines is even better)

Comment (liba2k) [reply qiling/os/uefi/utils.py:29]: I'm sure I did, maybe my editor isn't setup correctly and removed.

**Comment by aquynh** [comment]
btw, please document how to use `gen_hooks.py`

File: qiling/os/uefi/utils.py (hunk 1 above)
Comment (liba2k) [reply qiling/os/uefi/utils.py:29]: Done.

**Comment by liba2k** [comment]
gen_hooks.py is a little out of date now. It was used to generate the runtime and bootup files.
I think we can just delete it from the project.

**Comment by aquynh** [comment]
cool, then please delete that file in this PR too.

File: qiling/os/uefi/utils.py [hunk 2]
```diff
@@ -18,4 +20,21 @@ def check_and_notify_protocols(ql):
...
+def write_int64(ql, address, num):
+    if ql.archendian == QL_ENDIAN.EL:
+        ql.mem.write(address, struct.pack('<Q',(num)))
+    else:
+        ql.mem.write(address, struct.pack('>Q',(num)))
```
Comment (aquynh) [inline qiling/os/uefi/utils.py:34]: blank lines here, too.

Comment (liba2k) [reply qiling/os/uefi/utils.py:35]: Done.

**Comment by xwings** [comment]
I will merge and fix it.

**Pull request merged**